Solutions/DNS Domain Solution/Data/Solution_DNS.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "DNS Domain Solution for Microsoft Sentinel",
+  "Author": "Microsoft - support@microsoft.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Azure_Sentinel.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "This is a [domain solution](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fazure%2Fsentinel%2Fsentinel-solutions-catalog%23domain-solutions&data=05%7C01%7Ckavishbakshi%40microsoft.com%7Cbe2a496082b24caa4b8c08da9cefacca%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637994850502413731%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OJegu%2B2EqD7rmYmK9pm9QniD6YWp5ooloZ6tHzcwVi0%3D&reserved=0) and does not include any data connectors. The content in this solution requires one of the product solutions below , as well as any other connector or data source normalized to the [ASIM](https://aka.ms/AboutASIM).\n\n**Prerequisite :-**\n\n Install one or more of the listed solutions, or develop your custom ASIM parsers to unlock the value provided by this solution.\n 1. Windows Server DNS \n 2. Azure Firewall \n 3. Cisco Umbrella \n 4. Corelight Zeek \n 5. Google Cloud Platform DNS \n 6. Infoblox NIOS \n 7. ISC Bind \n 8. Vectra AI \n 9. Zscaler Internet Access \n\n**Underlying Microsoft Technologies used:** \n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in Preview state or might result in additional ingestion or operational costs: \n 1. Product solutions as described above \n 2. Logic app for data summarization\n\n**Recommendation :-**\n\nIt is highly recommended to use the **Summarize data** logic app playbook provided with this solution as it will significantly improve the performance of the Workbook, Analytic rules & Hunting queries.",
+  "Workbooks": ["Workbooks/DNSSolutionWorkbook.json"],
+  "Analytic Rules": [
+                    "Analytic Rules/DNSRequestToMaliciousDomain.yaml",
+                    "Analytic Rules/ExcessiveNXDOMAINDNSQueriesAnomalyBased.yaml",
+                    "Analytic Rules/ExcessiveNXDOMAINDNSQueriesStaticThresholdBased.yaml",
+                    "Analytic Rules/MultipleErrorsReportedForSameDNSQueryAnomalyBased.yaml",
+                    "Analytic Rules/MultipleErrorsReportedForSameDNSQueryStaticThresholdBased.yaml",
+                    "Analytic Rules/PotentialDGADetectedviaRepetitiveFailuresAnomalyBased.yaml",
+                    "Analytic Rules/PotentialDGADetectedviaRepetitiveFailuresStaticThresholdBased.yaml",
+                    "Analytic Rules/RareClientObservedWithHighReverseDNSLookupCountAnomalyBased.yaml"
```

Comment:
@vakohl, please add comma (,) at end of this line, thanks.

Reply:
done